### ISSUE TITLE
[BUG] "Powered by" insufficient color contrast

### ISSUE THREAD

**Issue by jonas-hoebenreich** [formbricks/formbricks#3691]
### Issue Summary

The "Powered by" section has an insufficient color contrast of just 2.56 (WCAG 2 AA minimum contrast ratio threshold: 4.5:1)

### Steps to Reproduce

1. Check color contrast of "Powered By Text"

### Expected behavior

Text has sufficient color contrast.

### Screenshots

![image](https://github.com/user-attachments/assets/690a38f3-bba9-40cf-bb2e-d3ab9218bb83)

### Environment

- [X] Formbricks Cloud (app.formbricks.com)
- [ ] Self-hosted Formbricks

### Desktop (please complete the following information)

```markdown
- OS:
- Node:
- npm:
```

**Comment by jonas-hoebenreich** [comment]
I can fix this /assign

**Comment by jobenjada** [comment]
/award 50 for spotting and fixing (please do it in 1 PR with the other issue)
